upup/pkg/fi/cloudup/awstasks/subnet.go

```diff
@@ -196,6 +205,22 @@ func (_ *Subnet) RenderAWS(t *awsup.AWSAPITarget, a, e, changes *Subnet) error {
 		}
 	}
 
+	if e.IPv6CIDR != nil && strings.HasPrefix(aws.StringValue(e.IPv6CIDR), "/") {
+		vpcIPv6CIDR, err := findVPCIPv6CIDR(t.Cloud, e.VPC.ID)
+		if err != nil {
+			return err
+		}
+		if vpcIPv6CIDR == nil {
+			return fi.NewTryAgainLaterError("waiting for the VPC IPv6 CIDR to be assigned")
```

Comment:
It's also possible it's a shared VPC with no IPv6 allocation forthcoming.

Reply:
In this case, there will be an error from `VPCAmazonIPv6CIDRBlock` that says: "IPv6 CIDR block provided by Amazon not found".